ipalib/install/certstore.py

```diff
@@ -29,6 +30,31 @@
 from ipalib import errors, x509
 from ipalib.constants import IPA_CA_CN
 
+# get_ca_certs() used to return a tuple with four elements.
+# The CA cert info object is a named tuple with same four elements and
+# additional fields.
+
+
+class CACertInfo(typing.NamedTuple):
+    """CA certificate information object
+    """
+    cert: x509.IPACertificate
+    nickname: str
+    trusted: typing.Optional[bool] = None
+    ext_key_usage: typing.Optional[list] = None
+
+    @property
+    def trustflags(self) -> TrustFlags:
+        return key_policy_to_trust_flags(
+            trusted=self.trusted,
+            ca=self.ca,
+            ext_key_usage=self.ext_key_usage
+        )
+
+    @property
+    def ca(self) -> bool:
+        return True
```

Comment:
What is the purpose of this property if it is always True? Future-proofing?

Reply:
Yes, it was initially for future-proofing. It's not necessary and I removed the property again.